content/en/tracing/services/service_naming_rules.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Service naming rules
+---
+
+{{< callout url=false
+ btn_hidden="true" header="Join the Preview!">}}
+Service naming rules is in Preview.
+{{< /callout >}}
+
+{{< site-region region="gov" >}}<div class="alert alert-danger"> Service naming rules is unavailable in the selected site ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Update how your services appear across Datadog without changing tracer configuration or redeploying code. Service naming rules allows you to rename, merge, or split services, or create new services based on infrastructure tags from the Datadog UI. You can also create naming rules for other entity types, such as datastores and queues.
+
+<div class="alert alert-info">Each organization can contain up to 100 naming rules.</div>
+
+## Prerequisites
+
+You must have the `apm_service_renaming_write` permission to create renaming rules. See [Permissions][1] for details on Datadog role-based access control.
+
+### Tracer version requirements
+
+You can create service naming rules only for services instrumented with supported tracer versions. If a service is reporting from an older tracer version, upgrade the tracer before creating naming rules for that service.
+
+| Language   | Minimum supported tracer version |
+|------------|----------------------------------|
+| Java       | [1.20][2]                        |
+| Dotnet     | [3.4][3]                         |
+| JavaScript | [4.16][4]                        |
+| Python     | [1.19][5]                        |
+| Go         | [1.55][6]                        |
+| PHP        | [0.94][7]                        |
+| Ruby       | [1.15][8]                        |
+
+## Create a service naming rule 
+
+### Step 1: Select naming action and entities to target
+
+1. In Datadog, navigate to **APM > Software Catalog > Manage > Manage Renaming Rules** and click **+ Add Rule**. 
+
+   Alternatively, navigate to **APM > Software Catalog** and click on a service to open the service side panel. From there, click **Service Page > Service Renaming**.
+
+   {{< img src="tracing/services/renaming_rules/service-side-panel.png" alt="The side panel for a particular service, showing the Service Page dropdown menu with a Service Renaming option" style="width:100%;" >}}
+
+1. Choose a naming action you want to perform for your new naming rule.
+   - You can select to split a single entity, rename an entity, merge multiple entities together, or rename several entities.
+   - You can also choose to identify a service based on infrastructure tags.
+
+1. Use the search bar to select the entities you want to rename.
+   - You can select one or more entities, but all must be of the same type (service, datastore, or queue).
+   - As you select entities, a span query is built in the background. To edit the query, select **Build Advanced Query**.
+   - If you're correlating a service with infrastructure tags, you can only select **one** service.
```

Comment:
I'd use italics here, since we're already using bold for UI labels.
```suggestion
   - If you're correlating a service with infrastructure tags, you can only select _one_ service.
```